zerver/tests/test_templates.py

```diff
@@ -97,6 +97,9 @@ def test_templates(self):
             'zilencer/enterprise_tos_accept_body.txt',
             'zerver/zulipchat_migration_tos.html',
             'zilencer/enterprise_tos_accept_body.txt',
+            'zerver/help/index.md',
```

Comment:
Due to recent changes, we should no longer need these here.  Can you remove them?